go.sum

```diff
@@ -434,6 +434,8 @@ github.com/tendermint/iavl v0.14.0 h1:Jkff+IFrXxRWtH9Jn/ga/2cxNnzMTv58xEKgCJsKUB
 github.com/tendermint/iavl v0.14.0/go.mod h1:QmfViflFiXzxKLQE4tAUuWQHq+RSuQFxablW5oJZ6sE=
 github.com/tendermint/tendermint v0.33.5 h1:jYgRd9ImkzA9iOyhpmgreYsqSB6tpDa6/rXYPb8HKE8=
 github.com/tendermint/tendermint v0.33.5/go.mod h1:0yUs9eIuuDq07nQql9BmI30FtYGcEC60Tu5JzB5IezM=
```

Comment:
should these versions be pruned? `go mod tidy` maybe?

Reply:
it didn't do anything 🤷‍♂️

Reply:
oh I think it is because tendermint/iavl relies on tendermint v0.33.5?

Reply:
> oh I think it is because tendermint/iavl relies on tendermint v0.33.5?

That's a bit unfortunate. @fedekunze how about making a iavl release that fixes that? @marbar3778